crates/terminal_view/src/terminal_view.rs

```diff
@@ -177,8 +177,8 @@ impl TerminalView {
             Event::NewNavigationTarget(maybe_navigation_target) => {
                 this.can_navigate_to_selected_word = match maybe_navigation_target {
                     Some(MaybeNavigationTarget::Url(_)) => true,
-                    Some(MaybeNavigationTarget::PathLike(maybe_path)) => {
-                        !possible_open_targets(&workspace, maybe_path, cx).is_empty()
+                    Some(MaybeNavigationTarget::PathLike(maybe_path, cwd)) => {
+                        !possible_open_targets(&workspace, cwd, maybe_path, cx).is_empty()
```

Comment:
This starts to be dangerous with the new code: `terminal` usually has some `cwd`, so now we would always highlight any relative path due to that bit of code.

Yet, we still may not be able to navigate to that path, because it's non-existing, so we will confuse people.
Previous code was not also good with that bit, but the new code, due to terminal process almost always having some `cwd` when its output is being cmd-hovered, will highlight more false-positives.